src/Uno.UI/UI/Xaml/Controls/ScrollViewer/ScrollViewer.cs

```diff
@@ -634,6 +634,11 @@ public double HorizontalOffset
 		/// </summary>
 		internal bool ComputedIsVerticalScrollEnabled { get; private set; } = false;
 
+
+		internal double MinHorizontalOffset => 0;
+
+		public double MinVerticalOffset => 0;
```

Comment:
```suggestion
		inetrnal double MinVerticalOffset => 0;
```

Not part of the UWP API https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.scrollviewer?view=winrt-20348 (while `MinZoomFactor` is ... don't ask me why)

Reply:
Indeed, I changed `MinHorizontalOffset` but somehow missed this one :D